packages/schema-generator/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@aws-amplify/schema-generator",
```

Comment:
what's the long-term ownership plan for this package? Should this be in the amplify-backend repo or in a data-team-owned repo?

Reply:
The main logic exists in API repo's `@aws-amplify/graphql-schema-generator` package. This package is a wrapper on top of that. I noticed this pattern for `model-generator` and `form-generator` and used the same.